samples/AspNetCore/MyWebhookEventProcessor.cs

```diff
@@ -14,7 +15,7 @@ public MyWebhookEventProcessor(ILogger<MyWebhookEventProcessor> logger)
             this.logger = logger;
         }
 
-        protected override void ProcessPullRequestWebhook(WebhookHeaders headers, PullRequestEvent pullRequestEvent, PullRequestAction action)
```

Comment:
Can we make this an async example? Even something like `await Task.Delay(1000);`

Reply:
Have added this below